### PR TITLE
Extend unit test coverage for WC_Payment_Token_ACSS and WC_Payment_Token_Becs_Debit

### PR DESCRIPTION
**Pull request by noruzzamans**
## Changes proposed in this Pull Request:

Extend unit test coverage for the `WC_Payment_Token_ACSS` and `WC_Payment_Token_Becs_Debit` classes.

Following the pattern established in the token test suites, this PR extends coverage for missing edge cases, incomplete payment method payloads, and aligns `setUp()` method visibility:

- `tests/phpunit/payment-tokens/class-wc-payment-token-acss-test.php`:
  - Token type verification (`WC_Stripe_Payment_Methods::ACSS_DEBIT`)
  - Localized display name formatting with bank name and last 4 digits (`get_display_name()`)
  - Display name formatting with partial or empty values via data provider
  - Token validation constraints (`validate()`) via data provider
  - Getter and setter pairs (`token`, `fingerprint`, `bank_name`, `last4`, `payment_method_type`) via data provider
  - Payment method equality comparisons (`is_equal_payment_method()`) covering type and fingerprint matching, mismatched type, mismatched fingerprint, empty fingerprint matching, missing `acss_debit` property, and missing fingerprint in `acss_debit`
  - Method visibility and return type normalization (`public function setUp(): void`)

- `tests/phpunit/payment-tokens/class-wc-payment-token-becs-debit-test.php`:
  - Token type verification (`WC_Stripe_Payment_Methods::BECS_DEBIT`)
  - Localized display name formatting with last 4 digits and empty last 4 (`get_display_name()`)
  - Token validation constraints (`validate()`) via data provider
  - Getter and setter pairs (`last4`, `payment_method_type`, `fingerprint`) via data provider
  - Payment method equality comparisons (`is_equal_payment_method()`) covering type and fingerprint matching against `au_becs_debit`, mismatched type, mismatched fingerprint, empty fingerprint matching, missing `au_becs_debit` property, and missing fingerprint in `au_becs_debit`
  - Method visibility and return type normalization (`public function setUp(): void`)

## Testing instructions

Verify static analysis, linting, and automated tests:

1. Run PHPCS:
```bash
./vendor/bin/phpcs --standard=phpcs.xml.dist tests/phpunit/payment-tokens/class-wc-payment-token-acss-test.php tests/phpunit/payment-tokens/class-wc-payment-token-becs-debit-test.php
```

2. Run PHPStan:
```bash
vendor/bin/phpstan analyse --configuration=phpstan.neon tests/phpunit/payment-tokens/class-wc-payment-token-acss-test.php tests/phpunit/payment-tokens/class-wc-payment-token-becs-debit-test.php --memory-limit=2G
```

3. Run the PHPUnit tests in the test environment:
```bash
npm run test:php -- --filter="WC_Payment_Token_ACSS_Test|WC_Payment_Token_Becs_Debit_Test"
```

---

- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### Changelog entry

- [x] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Comment</summary>

#### Comment
Test-only addition extending unit test coverage for `WC_Payment_Token_ACSS` and `WC_Payment_Token_Becs_Debit`.

</details>

### CODE REVIEW

File: tests/phpunit/payment-tokens/class-wc-payment-token-becs-debit-test.php [hunk 1]
```diff
@@ -45,38 +73,60 @@ public function test_validate( array $fields, bool $expected ) {
...
+	/**
+	 * Data provider for `test_getters_setters`.
+	 *
+	 * @return array
+	 */
+	public function provide_test_getters_setters(): array {
+		return [
+			'token'       => [ 'set_token', 'get_token', 'pm_test_becs_123', 'The token property should match the value that was set.' ],
+			'last4'       => [ 'set_last4', 'get_last4', '4356', 'The last4 property should match the value that was set.' ],
+			'fingerprint' => [ 'set_fingerprint', 'get_fingerprint', 'test_fp_becs_123', 'The fingerprint property should match the value that was set.' ],
```
[the remark this answers is not in the document]
Comment (noruzzamans) [reply tests/phpunit/payment-tokens/class-wc-payment-token-becs-debit-test.php:112]: @coderabbitai The `WC_Payment_Token_Becs_Debit` class does not implement `set_payment_method_type()` or `get_payment_method_type()` (unlike ACSS or BACS). Calling these undefined methods results in a fatal error (`Error: Call to undefined method WC_Payment_Token_Becs_Debit::set_payment_method_type()`), which is why it was intentionally removed from the getter/setter test matrix.

File: tests/phpunit/payment-tokens/class-wc-payment-token-acss-test.php [hunk 2]
```diff
@@ -77,8 +155,73 @@ public function test_is_equal_payment_method( string $token_fingerprint, bool $e
...
+			'empty token fingerprint'           => [
+				'',
+				(object) [
+					'type'                                => WC_Stripe_Payment_Methods::ACSS_DEBIT,
+					WC_Stripe_Payment_Methods::ACSS_DEBIT => (object) [
+						'fingerprint' => '',
+						'last4'       => '9876',
+					],
+				],
+				true,
```
[the remark this answers is not in the document]
Comment (noruzzamans) [reply tests/phpunit/payment-tokens/class-wc-payment-token-acss-test.php:203]: @coderabbitai This PR is strictly a 100% test-only contribution aimed at documenting existing behavior without production modifications. In `woocommerce-gateway-stripe`, all payment token classes (`ACSS`, `BACS`, `BECS`, `Card`, `SEPA`) implement `is_equal_payment_method()` using strict equality on the fingerprint property (`=== $this->get_fingerprint()`). The test accurately verifies this existing contract (consistent with the merged `BACS` token tests in #5959). Altering the production equality logic to reject empty fingerprints would create behavioral divergence across token classes and belongs in a separate dedicated core PR if maintainers choose to pursue it.

**Review by daledupreez — APPROVED** [review]
Thanks for the tests, @noruzzamans! These are looking good to me.
